include/swift/AST/DiagnosticsSema.def

```diff
@@ -37,15 +37,20 @@
   DIAG(NOTE,ID,Options,Text,Signature)
 #endif
 
-
-NOTE(type_declared_here,none,
-     "type declared here", ())
+NOTE(kind_declname_declared_here,none,
+      "%0 %1 declared here", (DescriptiveDeclKind, DeclName))
```

Comment:
Indentation is off here and other new definitions :(

Reply:
That's intentional, I changed it to the indentation of the rest of the file. It was this small chunk that had 5 spaces instead of 6.

Reply:
I just wanted to point out that `”` has to be aligned with first letter of the name like the rest of the file...

Reply:
Of course, my bad.